### PR TITLE
Convert/upgrade to latest Arcade

### PR DESCRIPTION
**Pull request by mikem8361**
Upgrade Arcade to version 1.0.0-beta.19412.7

Issues addressed:

https://github.com/dotnet/symstore/issues/108
https://github.com/dotnet/symstore/issues/104
https://github.com/dotnet/symstore/issues/91
https://github.com/dotnet/symstore/issues/90
https://github.com/dotnet/symstore/issues/89

### CODE REVIEW

File: src/Directory.Build.targets [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (hoyosjs) [inline src/Directory.Build.targets]: Why isn't this on the root level?

File: src/dotnet-symbol/dotnet-symbol.csproj [hunk 2]
```diff
@@ -1,23 +1,20 @@
 ﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Project Sdk="RoslynTools.RepoToolset">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <RuntimeFrameworkVersion>2.1.0</RuntimeFrameworkVersion>
```
Comment (hoyosjs) [inline src/dotnet-symbol/dotnet-symbol.csproj]: Please remove the `RuntimeFrameworkVersion`. This is not a good idea to embed unless you have very good reasons for it.

File: src/dotnet-symbol/dotnet-symbol.csproj [hunk 3]
```diff
@@ -1,23 +1,20 @@
...
+    <ToolCommandName>dotnet-symbol</ToolCommandName>
+    <RootNamespace>dotnet.symbol</RootNamespace>
     <IsPackable>true</IsPackable>
     <PackAsTool>true</PackAsTool>
     <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;osx-x64</PackAsToolShimRuntimeIdentifiers>
-    <!-- The package version needs to be hard coded as a stable version so "dotnet tool install -g dotnet-symbols" works -->
-    <Version>1.0.5</Version>
-    <PackageVersion>$(Version)</PackageVersion>
-    <ToolCommandName>dotnet-symbol</ToolCommandName>
+    <PackagedShimOutputRootDirectory>$(OutputPath)</PackagedShimOutputRootDirectory>
```
Comment (hoyosjs) [inline src/dotnet-symbol/dotnet-symbol.csproj]: What is this used by?

Comment (mikem8361) [reply src/dotnet-symbol/dotnet-symbol.csproj]: This is so the shim gets properly signed. Our global tools in diagnostics also do this.

File: src/dotnet-symbol/dotnet-symbol.csproj (hunk 2 above)
Comment (mikem8361) [reply src/dotnet-symbol/dotnet-symbol.csproj]: I had a good reason with I first wrote this. I don't remember now.  I guess I'll remove it and see what breaks (if anything).

File: License.txt [hunk 4]
```diff
@@ -1,15 +1,23 @@
-Copyright (c) .NET Foundation and Contributors
+The MIT License (MIT)
```
[the remark this answers is not in the document]
Comment (mikem8361) [reply License.txt]: To match the other repos. I don't know why symstore original had the apache license txt.

**Pull request merged**